src/component/mxgraph/MxGraphRenderer.ts

```diff
@@ -85,7 +85,7 @@ export default class MxGraphRenderer {
   }
 
   computeStyle(bpmnCell: Shape | Edge, labelBounds?: Bounds): string {
-    const styleValues = new Map<string, string | number | boolean>();
+    const styleValues = new Map<string, string | number>();
```

Comment:
Why is it possible to have number, and not boolean ?

Reply:
This is possible but using boolean was the cause of wrong rendering of non interrupting event, see comment in 15bcd34
I could be possible to use boolean but in that case, we have to update the filter function when creating the string style: for number equal to 0, we should have added code to ensure that font style is not part of the style